analyze/views.py

```diff
@@ -26,29 +26,44 @@ class AnalyzeView(TemplateView):
     template_name = 'analyze/overview_files.html'
 
     def get_context_data(self, **kwargs):
-        item_list = AASPItem.objects.all()
-        context = {
-        'item_list': sorted(item_list, key=lambda x: x.item_id),
-        }
+        context = super().get_context_data(**kwargs)
+
+        print(self.request.GET.get('speaker'))
+
+        item_list = AASPItem.objects.all().order_by('-id')
+        speaker = self.request.GET.get('speaker')
+        if speaker is not None and speaker != 'all':
+            item_list = item_list.filter(speaker=speaker)
+
+        print(item_list)
+
```

Comment:
`print` statements should be removed before merging. I'll take care of that.

Reply:
Oh, yes of course. Forgot about that one